db/migrate/20221214140623_add_translated_alert_type_rating_content_version_email_title.rb

```diff
@@ -0,0 +1,7 @@
+class AddTranslatedAlertTypeRatingContentVersionEmailTitle < ActiveRecord::Migration[6.0]
+  def change
+    AlertTypeRatingContentVersion.all.each do |alert_type_rating_content_version|
+      alert_type_rating_content_version.update(email_title_en: alert_type_rating_content_version[:email_title])
+    end
+  end
+end
```

Comment:
I think we've previously done this sort of data migration in after_party tasks, like 20220629153807_copy_programme_type_fields_to_mobility.rake

(I think the general approach was to keep schema changes in the migrations, but data changes - especially with AR - in the after_party tasks).

Does this achieve the same thing?

Reply:
hi @jhigman yep does the same thing but I'll move to after_party tasks (email & sms fields could be updated in a single migration too)

Reply:
hi @jhigman I've added the after_party task and removed the migrations.  would you be able to give another review when you get a mo please 🙏   thanks!